langchain_dappier/retrievers.py

```
@@ -0,0 +1,212 @@
+    num_articles_ref: int = 0
+    """Minimum number of articles from the ref domain specified.
+    The rest will come from other sites within the RAG model."""
+    search_algorithm: Literal[
+        "most_recent", "most_recent_semantic", "semantic", "trending"
+    ] = "most_recent"
+    """Search algorithm for retrieving articles."""
+    api_key: Optional[str] = None
+    """The API key used to interact with the Dappier APIs."""
+
+    def _get_relevant_documents(
+        self, query: str, *, run_manager: CallbackManagerForRetrieverRun, **kwargs: Any
```

Ruff (0.8.2)

Unused method argument: `run_manager`

(ARG002)